brainiak/utils/fmrisim.py

```diff
@@ -571,6 +579,9 @@ def double_gamma_hrf(stimfunction,
         scale_function : bool
             Do you want to scale the function to a range of 1
 
+        temporal_resolution : float
+          What is the temporal resolution (in s) of the stimulus function.
+          In other words, how many elements per second?
```

Comment:
I think the note taken from the old comment, `(in s)`, is misleading here. Why not have the same description for `temporal_resolution` as in the other functions?